src/domain/website.ts

```diff
@@ -613,6 +613,7 @@ export class WebsiteManager implements EntityManager<Website, AddWebsite> {
               message_id: volumeId,
               type: cv.target,
               updated_at: new Date().toISOString(),
+              options: { catch_all_path: '/404.html' },
```

Comment:
Why are we handling domain aggregates from here? This should be managed by the domain class not by the website class. We should invoke the proper method of the domain class from here